src/fsharp/LexFilter.fs

```diff
@@ -394,26 +394,19 @@ let parenTokensBalance t1 t2 =
     | _ -> false
     
 /// Used to save some aspects of the lexbuffer state
-[<Struct>]
 type LexbufState(startPos: Position, 
                  endPos  : Position, 
                  pastEOF : bool) = 
     member x.StartPos = startPos
     member x.EndPos = endPos
     member x.PastEOF = pastEOF
-
-[<Struct>]
-type PositionTuple =
-    val X: Position
-    val Y: Position
-    new (x: Position, y: Position) = { X = x; Y = y }
-
```

Comment:
After looking at what changed below, getting rid of this is great. `X` was never used, is that correct? So the whole point of `PositionTuple` was just to use `Y`.

Reply:
Yeah, I noticed that it was only ever being used to find the end position of that lastToken, so the start token position was wasted space. I tested with having only an AbsoluteOffset but due to the fact that these carry over lines, you need the LinePosition data also.